framer/Animation.coffee

```diff
@@ -417,7 +403,7 @@ class exports.Animation extends BaseClass
 		return _.pick(@layer, _.keys(@properties))
 
 	@isAnimatable = (v) ->
-		_.isNumber(v) or _.isFunction(v) or isRelativeProperty(v) or Color.isColorObject(v) or Gradient.isGradientObject(v) or SVGPath.isPath(v)
+		_.isNumber(v) or _.isFunction(v) or isRelativeProperty(v) or Color.isColorObject(v) or Gradient.isGradientObject(v) #or SVGPath.isPath(v)
```

Comment:
Remove commented out code.

Reply:
Yeah it should be enabled again, but I had some file-inheritance/require issues that I solved by disabling this alltogether